plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -414,25 +423,27 @@ func (s *SpanReader) executeQuery(span opentracing.Span, query cassandra.Query,
 	tableMetrics.Emit(err, time.Since(start))
 	if err != nil {
 		logErrorToSpan(span, err)
-		span.LogFields(otlog.String("query", query.String()))
+		span.SetAttributes(attribute.Key("query").String(query.String()))
```

Comment:
I am not sure this is needed. `executeQuery` is typically called after `startSpanForQuery`, which already logs the query to the span